src/convertToObject.js

```diff
@@ -6,7 +6,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  if (sourceString.length === 0) {
+    return {};
+  }
+
+  const cleanedSourceString = sourceString.replace(/\s*;\s*/g, ';').trim();
+  const strArray = cleanedSourceString.split(';');
```

Comment:
The variable name `strArray` could be more descriptive. Consider naming it something like `styleDeclarations` to reflect what it actually contains.